docker-working-env-base.yaml

```diff
@@ -0,0 +1,125 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: This template deploys a VPC and an EC2 instance with Docker already installed, with the goal of standardizing the experience of the users for the Docker and k8s workshops.
+
+Parameters:
+  KeyPair:
+    Type: AWS::EC2::KeyPair::KeyName
+    Description: The EC2 key pair to be used for SSHing into the instance
+
+Mappings:
```

Comment:
I think we should use the SSM lookup method here to keep the template short and prevent the need for regular updates. You can see an example (for Amazon Linux 1) here: https://github.com/awslabs/amazon-ecs-nodejs-microservices/pull/8/commits/01e2615f7cff972c96b23921fb7b3a878dbf42f5